gradio/blocks.py

```diff
@@ -1260,6 +1260,14 @@ def preprocess_data(
                     inputs_cached = processing_utils.move_files_to_cache(
                         inputs[i], block
                     )
+                    if getattr(block, "data_model", None) and inputs_cached is not None:
+                        if issubclass(block.data_model, GradioModel):  # type: ignore
+                            print("block.data_model", block.data_model, block)
```

Comment:
Can we get rid of the print statements?

Reply:
Oops I'll do it on @dawoodkhan82's branch, thanks for catching

Reply:
Done here: https://github.com/gradio-app/gradio/pull/6137

Reply:
Let me know if you find anything else! I merged this just now so that we could have everything in place and do another release